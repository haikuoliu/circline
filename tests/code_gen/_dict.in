dict<int> d_int = {1: 11, 2: 22, 3: 33};
print(d_int);
print(d_int.get(1));
print(d_int.put(4, 44));
print(d_int.remove(2));
print(d_int.size());
list<int> l_int = d_int.keys();
print(l_int);
print(d_int.has(2));
print(d_int.has(3));
