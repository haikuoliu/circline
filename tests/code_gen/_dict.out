{2: 22, 1: 11, 3: 33}
11
{2: 22, 1: 11, 3: 33, 4: 44}
{1: 11, 3: 33, 4: 44}
3
list:[1, 3, 4]
false
true
